src/components/SettingsModal/SettingsModal.scss

```diff
@@ -0,0 +1,12 @@
+.modal-content, .modal-card{
+    max-width: 80%;
+}
+
+p{
+    margin: 0px !important;
+}
+
+label{
+    width: 100% !important;
+    font-size: 1em !important;
+}
```

Comment:
So ideally, we should try and avoid using `!important` for the styling. 

You can change [Typography](https://bulma.io/documentation/helpers/typography-helpers/) properties with Bulma, as well as [Spacing](https://bulma.io/documentation/helpers/spacing-helpers/) stuff